services/r2ctl/main/main.go

```diff
@@ -108,12 +109,20 @@ func main() {
 		"service-name": "r2",
 	})
 	r2ServiceInstrumentOpts := instrumentOpts.SetMetricsScope(r2ServiceScope)
+	var propagationDelay time.Duration
```

Comment:
Prior to this diff we were not using the MutableRuleSet object to update entire rule sets. But it sounds like this is going to be moved to the m3metrics repo so this will eventually go away.